src/Raven.Server/Documents/TcpHandlers/SubscriptionConnection.cs

```diff
@@ -485,6 +485,10 @@ private async Task<SubscriptionConnectionClientMessage> GetReplyFromClientAsync(
                     BlittableJsonDocumentBuilder.UsageMode.None,
                     _copiedBuffer.Buffer))
                 {
+                    if (blittable.TryGet("ChangeVector", out string cv))
+                    {
+                        TcpConnection.lastEtagReceived = cv.ToChangeVector()[0].Etag;
```

Comment:
This is likely wrong.
Better to have `LastChangeVector`

Reply:
@efratshenhar this is used to identify the batches?

Reply:
yes

Reply:
@ayende I think it is easier to match between source and destination via etag than CV. Thoughts?

Reply:
@efratshenhar it probably requires Studio work in Replication Stats view (Ongoing Stats view)?

Reply:
that may be the case, sure, but the code we have now is not going to work.
It is saying, give me the first cv's etag. What if we have multiple nodes?

Reply:
So maybe it would be easier via some internal identifier?

E.g.
```
private static long _batchId = 1;

public static long GetNextBatchId()
{
    return Interlocked.Increment(ref _batchId);
}
```

Reply:
How would that help the user seeing this? What does this corrleates to?

Reply:
on A node I will see an Outgoing batch with Id 10 and on B node in Incoming I can identify it easily.

Reply:
This is to see where you are on the subscription. No? In that case, etag isn't meaningful.
And an arbitrary integer won't help, you can't compare between nodes, there is only one node in subscription.
Or did I miss something?

Reply:
This one is in subscription, you are correct, but the original task (in YT) was to match replication batches

Reply:
For replication, the etag is already handled, so no issue. 
For subscription, I wonder if we even need it, we already report the last change vector, IIRC

Reply:
> @efratshenhar it probably requires Studio work in Replication Stats view (Ongoing Stats view)?

We can see the Etag but not the bytes sent/received